lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -1055,6 +1055,21 @@ def test_query_counts(self, (sql_calls, mongo_calls, self_paced), self_paced_ena
             resp = views.progress(self.request, course_id=unicode(self.course.id))
         self.assertEqual(resp.status_code, 200)
 
+    @patch('courseware.grades.grade', Mock(return_value={'grade': 'Pass', 'percent': 0.75, 'section_breakdown': [],
```

Comment:
either break the long lines according to http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#breaking-long-lines, or set the return value later.